src/components/ModalPin.js

```diff
@@ -79,9 +77,18 @@ export class ModalPin extends React.Component {
 
     // Set the PIN on the instance variable and close the modal.
     this.pin = pin;
+    this.onSuccess();
```

Comment:
suggestion: Replace this with `this.onModalHidden()`. The logic will be the same, as I see it.

Reply:
I agree, thanks!